src/eager/setup.py

```diff
@@ -16,6 +16,9 @@
 parser.add_argument('--additional_libs', default=None, type=str, help='Additional libraries to link against')
 parser.add_argument('--compiler_args', default=None, type=str, help='Additional compiler args to use')
 parser.add_argument('--skip_tests', action='store_true', help='Skips running unit tests as part of the build tests')
+parser.add_argument('--use_preinstalled_torch', action='store_true', help='Use pre-installed torch from the python environment')
+parser.add_argument('--ort_path', default=None, type=str, help='Use pre-built Onnxruntime from this path')
+parser.add_argument('--build_torch_wheel', action='store_true', help='Build PyTorch wheel')
```

Comment:
It seems by implementation that this flag implies --skip_tests? If so, maybe add this to the help message to let user know

Reply:
This flag doesn't necessarily skip tests. What happens is if src/eager is built as a wheel using `python setup.py bdist_wheel`, the logic skips tests.

Reply:
Discussed on another thread, will be adding logic to run tests during wheel file packaging as well